parser_test.go

```diff
@@ -207,48 +207,73 @@ func TestParseTapeFile(t *testing.T) {
 }
 
 func TestParseCtrl(t *testing.T) {
-	t.Run("should parse with multiple modifiers", func(t *testing.T) {
-		tape := "Ctrl+Shift+Alt+C"
-		l := NewLexer(tape)
-		p := NewParser(l)
-
-		cmd := p.parseCtrl()
-
-		expectedArgs := []string{"Shift", "Alt", "C"}
-		args := strings.Split(cmd.Args, " ")
+	tests := []struct {
+		name     string
+		tape     string
+		wantArgs []string
+		wantErr  bool
+	}{
+		{
+			name:     "should parse with multiple modifiers",
+			tape:     "Ctrl+Shift+Alt+C",
+			wantArgs: []string{"Shift", "Alt", "C"},
+			wantErr:  false,
+		},
+		{
+			name:     "should parse with multiple modifiers",
+			tape:     "Ctrl+Space",
+			wantArgs: []string{"Space"},
+			wantErr:  false,
+		},
+		{
+			name:     "should parse with multiple modifiers",
+			tape:     "Ctrl+Shift+Space",
+			wantArgs: []string{"Shift", "Space"},
+			wantErr:  false,
+		},
+		{
+			name:     "should parse with multiple modifiers",
+			tape:     "Ctrl+Space+Shift",
```

Comment:
I think this should be an error as Space is a key and should not come before a modifier.

Reply:
Done! Added in a new commit